test/unit/test-coin-parse-url.js

```diff
@@ -64,3 +65,27 @@ test('Should throw for invalid ZEC address', t => {
   } catch (error) { }
   t.true(spy.threw())
 })
+
+test('Should parse ADA address (Byron era)', t => {
+  const addresses = [
+    '37btjrVyb4KFTChjvZKjxBfUHjpEFz3CzNDyChYqSfCVLABzxKcW6JWWTYXd1ELD8cirXJ6osWpkN3dVqouRgFbLmua5gCWv8Pha1AQyPtwSeMkF9s', // Daedalus like address
+    'Ae2tdPwUPEZKmwoy3AU3cXb5Chnasj6mvVNxV1H11997q3VW5ihbSfQwGpm' // Yoroi like address
+  ]
+  
+  for (let addr of addresses) {
+    let parsed = parseADAAddress('main', addr)
+    t.is(parsed, addr)
+  }
+})
+
+test('Should recognize invalid ADA address', t => {
```

Comment:
I would either rename it to "Should recognize invaliad ADA addresses" as you test multiple addresses, but rather would create a separate test for each "invalid" case.

Reply:
OK, will do separate tests :)